src/components/payment-table.tsx

```diff
@@ -99,7 +99,7 @@ export const columns: ColumnDef<Payment>[] = [
     header: 'Amount',
     cell: ({ row }) =>
       getAmountWithCurrencySymbol(
-        row.getValue('amountInCrypto'),
+        row.getValue('amountInCrypto') || row.getValue('amount'),
```

Comment:
Please apply this logic to the feeAmount too :pray: 
Please use the same `getAmountWithCurrencySymbol()` for the `feeAmount`